src/libstd/fs.rs

```diff
@@ -413,6 +414,9 @@ impl OpenOptions {
     /// This option, when true, will indicate that the file should be
     /// `write`-able if opened.
     ///
+    /// If a file already exist, the contents of that file get overwritten, but it is
+    /// not truncated.
```

Comment:
Could this clarify that the contents don't get immediately overwritten, only when an actual call to `write` is made? E.g. emphasize that the cursor is at the beginning so written data will tromp existing data.

Also as a side note could you make sure to format this comment and the ones below to 80 characters in width? (the style of the surrounding module)